_episodes/01-intro.md

```diff
@@ -212,7 +212,7 @@ stuck on it](../fig/python-sticky-note-variables-03.svg)
 
 > ## Sorting Out References
 >
-> What does the following program print out?
+> Python allows you to assign multiple values to multiple variables in one line by separating the variables and values with commas. What does the following program print out? What does the following program print out?
```

Comment:
I think this text could be helpful for users, since we did not cover this in the lesson content. It does not get into technical detail, but at this point in the lesson, I think that makes sense. Does this text work for you @maxim-belkin ?

One thing to note here @gerrydevine , where possible we try to keep lines in the markdown files to under 100 characters, so we would need to split this onto a second line.